rest/model/pod.go

```diff
@@ -41,6 +41,9 @@ type APICreatePod struct {
 
 // fromAPIEnvVars converts a slice of APIPodEnvVars to a map of environment variables and a map of secrets.
 func fromAPIEnvVars(api []*APIPodEnvVar) (envVars map[string]string, secrets map[string]string) {
+	envVars = make(map[string]string)
```

Comment:
Since this was only caught by tests, can you write a test for `(*APICreatePod).ToService`?